ui/dashboards/src/context/DashboardProvider/panel-editor-slice.ts

```diff
@@ -173,13 +175,15 @@ export function createPanelEditorSlice(): StateCreator<
     },
 
     openAddPanel(panelGroupId) {
-      // If a panel group isn't supplied, add to the first group
+      // If a panel group isn't supplied, add to the first group or create a group if there aren't any
+      let newGroup: PanelGroupDefinition | undefined = undefined;
       if (panelGroupId === undefined) {
-        const firstGroupId = get().panelGroupOrder[0];
-        if (firstGroupId === undefined) {
-          throw new Error('No panel groups to add a panel to');
+        panelGroupId = get().panelGroupOrder[0];
+        if (panelGroupId === undefined) {
+          newGroup = createEmptyPanelGroup();
+          newGroup.title = 'Panel Group';
```

Comment:
I like the idea of 'Group 1' but it's not my say in the end, just throwing it out there